### PR TITLE
Messaging functionality, bootstrap, layout updates

### PR DESCRIPTION
**Pull request by DilSab**
* Admin can post global messages that will be visible to users based on their type.
* Logged off users can see messages visible to all users.
* Teachers can see messages visible to all users and teachers.
* Admins can see all messages.
* Message counter is updated every 10 seconds.
###
* Added logger that is fired on message posted event. This logger writes to file so it requires Visual Studio to be run 'as administrator'.
###
* Keep in mind that messages disappear when their expiration date is up, though administrators can always see them.
###
* Had to update bootstrap since there were some important styles for responsiveness missing.

### CODE REVIEW

File: Model/GlobalMessage.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+namespace Model
+{
+    using System;
+    using System.Collections.Generic;
+    
+    public partial class GlobalMessage
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public int UserId { get; set; }
```
[the remark this answers is not in the document]
Comment (DilSab) [reply Model/GlobalMessage.cs]: Tik admin gali globalias žinutes siųsti.

File: ControllerProject/Logger/MessageLogger.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿using ControllerProject.Service;
+using System;
+using System.IO;
+
+namespace ControllerProject.Logger
+{
+    public class MessageLogger : BaseLogger, IMessageLogger
+    {
+        private string filePath = "MoodDetector\\Logs\\MoodDetectorLogs.txt";
```
Comment (agosu) [inline ControllerProject/Logger/MessageLogger.cs]: Pas mane nesusikuria arba nerandu

File: MoodDetectorWebApp/Controllers/LoginController.cs [hunk 3]
```diff
@@ -44,18 +41,18 @@ public ActionResult Login(LoginModel model)
                 {
                     case "Admin":                        
                         FormsAuthentication.SetAuthCookie(model.UserName, false);                       
-                        LoginController.AccessRights = "admin";
+                        AccessRights = "admin";
```
Comment (agosu) [inline MoodDetectorWebApp/Controllers/LoginController.cs]: Ačiū už mano durno kodo pataisymą

File: MoodDetectorWebApp/Controllers/MoodController.cs [hunk 4]
```diff
@@ -38,12 +38,14 @@ public ActionResult SessionsList()
             foreach (int id in sessionsIds)
             {
                 Session session = _moodService.GetSession(id);
-                SessionSummary sessionSummary = new SessionSummary();
-                sessionSummary.id = id;
-                sessionSummary.date = session.DateTime;
-                sessionSummary.studentClass = session.Class;
-                sessionSummary.subject = session.Subject;
-                sessionSummary.comment = session.Comments;
+                SessionSummary sessionSummary = new SessionSummary
```
Comment (agosu) [inline MoodDetectorWebApp/Controllers/MoodController.cs]: Danke again

Comment (DilSab) [reply MoodDetectorWebApp/Controllers/MoodController.cs]: kein thema

**Pull request merged**